lib/srv/db/mysql/events.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mysql
+
+import (
+	"github.com/gravitational/teleport/api/types/events"
+	libevents "github.com/gravitational/teleport/lib/events"
+	"github.com/gravitational/teleport/lib/srv/db/common"
+	"github.com/gravitational/teleport/lib/srv/db/mysql/protocol"
+	"github.com/siddontang/go-mysql/mysql"
+)
+
+// makeStatementPrepareEvent creates an audit event for MySQL statement prepare
+// command.
+func makeStatementPrepareEvent(session *common.Session, packet *protocol.StatementPreparePacket) events.AuditEvent {
+	return &events.MySQLStatementPrepare{
+		Metadata: common.MakeEventMetadata(session,
+			libevents.DatabaseSessionMySQLStatementPrepareEvent,
+			libevents.MySQLStatementPrepareCode),
+		UserMetadata:     common.MakeUserMetadata(session),
+		SessionMetadata:  common.MakeSessionMetadata(session),
+		DatabaseMetadata: common.MakeDatabaseMetadata(session),
+		Query:            packet.Query(),
+	}
+}
+
+// makeStatementExecuteEvent creates an audit event for MySQL statement execute
+// command.
+func makeStatementExecuteEvent(session *common.Session, packet *protocol.StatementExecutePacket, parameterDefinitions []mysql.Field) events.AuditEvent {
```

Comment:
Looks like `parameterDefinitions` is unused and can be removed.
```suggestion
func makeStatementExecuteEvent(session *common.Session, packet *protocol.StatementExecutePacket) events.AuditEvent {
```